restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -636,7 +636,7 @@ public Response getDocPath(
           if (filters.isEmpty()) {
             node = documentService.getJsonAtPath(db, namespace, collection, id, path);
             if (node == null) {
-              return Response.noContent().build();
+              return Response.status(Response.Status.NOT_FOUND).build();
```

Comment:
this change influences the `getDoc` method as well, is this fine?

Reply:
Yep! that's intentional :)